panel/.eslintrc.js

```diff
@@ -1,5 +1,8 @@
 module.exports = {
 	extends: ["eslint:recommended", "plugin:vue/recommended", "prettier"],
+	parserOptions: {
+		ecmaVersion: 2021
+	},
```

Comment:
Not sure about this change. I have to use version 2021 to use null coalescing assignment operator `??=` in js.

Reply:
If this does not break anything else, I'm happy about it. We could definitely use this in more places.

Reply:
Is this automatically compiling backwards on build - or are we raising browser requirements with this?